internal/ewma/ewma.go

```diff
@@ -0,0 +1,53 @@
+// Package ewma implements exponentially weighted moving averages.
+package ewma
+
+// Copyright (c) 2013 VividCortex, Inc. All rights reserved.
+// Please see the LICENSE file for applicable license terms.
+
+//go:generate msgp -file $GOFILE
+
+const (
+	// By default, we average over a one-minute period, which means the average
+	// age of the metrics in the period is 30 seconds.
+	avgMetricAge float64 = 30.0
```

Comment:
> Do we know this is updated every second?
> 
> It seems this is updated only when replication completes and not every second.

I copied paste this external library to our internal code because I needed to generate a msgp for it.

Basically I tested this library before, I just wanted a way to forget older values for the metric, but yeah it is easier if we wrote this ourselves